### PR TITLE
feat: Enable wiki by default and put it behind a flag

### PR DESCRIPTION
**Pull request by amirtds**
Configuration Pull Request
---

Make sure that the following steps are done before merging

  - [x] @devops team member has commented with :+1:

### CODE REVIEW

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -339,6 +339,9 @@ EDXAPP_FEATURES:
   SHOW_HEADER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_HEADER_LANGUAGE_SELECTOR }}"
   SHOW_FOOTER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_FOOTER_LANGUAGE_SELECTOR }}"
 
+# Disable wiki by default
+EDXAPP_WIKI_ENABLED: false
```
Comment (OmarIthawi) [inline playbooks/roles/edxapp/defaults/main.yml]:
```suggestion
EDXAPP_WIKI_ENABLED: true
```

We need it for Tahoe. Disabling it for specific customers seems like a better way to go.

**Review by OmarIthawi — CHANGES_REQUESTED** [review]

File: playbooks/roles/edxapp/defaults/main.yml (hunk 1 above)
Comment (amirtds) [reply playbooks/roles/edxapp/defaults/main.yml]: @OmarIthawi Done

**Review by OmarIthawi — APPROVED** [review]

File: playbooks/roles/edxapp/defaults/main.yml (hunk 1 above)
Comment (OmarIthawi) [reply playbooks/roles/edxapp/defaults/main.yml]: Thank you!

File: playbooks/roles/edxapp/defaults/main.yml [hunk 2]
```diff
@@ -339,6 +339,9 @@ EDXAPP_FEATURES:
   SHOW_HEADER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_HEADER_LANGUAGE_SELECTOR }}"
   SHOW_FOOTER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_FOOTER_LANGUAGE_SELECTOR }}"
 
+# Disable wiki by default
```
Comment (OmarIthawi) [inline playbooks/roles/edxapp/defaults/main.yml]:
```suggestion
```

**Review by OmarIthawi — CHANGES_REQUESTED** [review]

File: playbooks/roles/edxapp/defaults/main.yml (hunk 2 above)
Comment (OmarIthawi) [reply playbooks/roles/edxapp/defaults/main.yml]: One more thing to go.

Comment (amirtds) [reply playbooks/roles/edxapp/defaults/main.yml]: whoops, taking care of it now

Comment (amirtds) [reply playbooks/roles/edxapp/defaults/main.yml]: @OmarIthawi done

**Review by OmarIthawi — APPROVED** [review]
:rocket:

**Review by thraxil — APPROVED** [review]

**Pull request merged**